common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -349,6 +349,28 @@ static boolean equals(byte[] bytes1, int startPos1, int endPos1, byte[] bytes2,
         return bytes1[startPos1] == bytes2[startPos2];
     }
 
+    static int hashCode(byte[] bytes, int startPos, int endPos, long prime) {
```

Comment:
Great suggestion BTW :)  Rushed to get through approval process and "safe" method was not so safe in this sense...take a look at the code now.